core/schema_utils.py

```diff
@@ -35,8 +35,8 @@
 from core.constants import constants
 from core.domain import expression_parser
 from core.domain import html_cleaner
+from core.domain import skill_domain
 from core.domain import user_domain
-
```

Comment:
Why do you remove this line?

Reply:
Because I doesn't wanted space between the two imports. 

Should i put that line back ?

Reply:
Yeah, I don't see the reason why to remove it, also I think it is against our linting.